varcode/variant_collection.py

```diff
@@ -92,58 +94,55 @@ def _clone_metadata(self, new_variants):
             variants=new_variants,
             original_filename=self.original_filename)
 
+    @memoize
+    def high_impact_effects(self, *args, **kwargs):
```

Comment:
I'm going to leave argument forwarding for now since the set of globally relevant parameters hasn't stabilized yet. I do agree that eventually it would be nice to have explicit parameters on each method. 

Changing the `raise_on_error` parameter into a `VariantCollection.raise_on_annotation_error` property sounds good to me, adding it as an issue: https://github.com/hammerlab/varcode/issues/36